src/vma/util/match.cpp

```diff
@@ -158,9 +158,9 @@ static void get_rule_str(struct use_family_rule *rule, char *buf, size_t len)
 	}
 
 	char addr_buf_first[MAX_ADDR_STR_LEN];
-	char ports_buf_first[16];
+	char ports_buf_first[17];
```

Comment:
AFAIU this buffer goes into get_address_port_rule_str where it prints %d-%d which if switched to %u-%u should be maximum 12 bytes long including term null. since sport,eport is maximum a 5 digits number. How can it get to 17?

Reply:
may be you are right and probably I fixed two different reported issues independently.

Reply:
[2021-11-22T14:09:21.558Z] /scrap/jenkins/workspace/LibVMA/src/vma/util/match.cpp:146:23: error: ‘sprintf’ may write a terminating nul past the end of the destination [-Werror=format-overflow=]

[2021-11-22T14:09:21.558Z]     sprintf(ports_buf, "%u-%u", rule->sport, rule->eport);

[2021-11-22T14:09:21.558Z]                        ^~~~~~~

[2021-11-22T14:09:21.558Z] /scrap/jenkins/workspace/LibVMA/src/vma/util/match.cpp:146:11: note: ‘sprintf’ output between 4 and 17 bytes into a destination of size 16

[2021-11-22T14:09:21.558Z]     sprintf(ports_buf, "%u-%u", rule->sport, rule->eport);